omp/preconditioner/isai_kernels.cpp

```diff
@@ -109,16 +109,21 @@ void generic_generate(std::shared_ptr<const DefaultExecutor> exec,
     const auto i_cols = inverse_mtx->get_const_col_idxs();
     auto i_vals = inverse_mtx->get_values();
 
+    auto num_threads = static_cast<size_type>(omp_get_max_threads());
+    // OpenMP seems to have issues copying the arrays, so we do it manually
```

Comment:
```suggestion
```

Reply:
Does this still need to be updated ?

Reply:
Yes, I just forgot to remove the comment :)